docs/containers.conf.5.md

```diff
@@ -540,6 +540,10 @@ reserving the pod's resources for the lifetime of the pod. By default container
 engines run a built-in container using the pause executable. If you want override
 specify an image to pull.
 
+**kube_generate_type**="pod"
+
+Default Kubernetes kind/specification of the kubernetes yaml generated with the `podman kube generate` command. The possible options are `pod` and `deployment`.
```

Comment:
Shouldn't we point to `man podman-kube-play` for supported options?

We don't validate/parse the input here, so there's a risk of the docs getting out of date over time.